src/components/ShopRegistration.vue

```diff
@@ -1,61 +1,52 @@
 <template>
 
 <div class="container">
-    <div v-if="loading" class="loading">
-      Loading...
-    </div>
+  <h2>Einen neuen Markt / ein neues Geschäft hinzufügen</h2>
 
-    <div v-if="error" class="error">
-      {{ error }}
-    </div>
-    <form method="POST" action="/api/shops" class="wb-3">
-        <h3 style="text-align: left">Anschrift</h3>
-        <div class="form-row mb-3" style="text-align: left; margin: 0 auto;">
-            <div class="col">
-                <label for="marketName">Names des Supermarktes</label>
-                <input id="marketName" class="form-control" v-model.trim="newShop.name" type="text" placeholder="Name des Supermarkts">
-            </div>
-        </div>
+  <div v-if="loading" class="loading">
+    Loading...
+  </div>
 
-        <div class="form-row mb-3" style="text-align: left; margin: 0 auto;">
-            <div class="col">
-                <label for="marketAddress">Adresse des Supermarktes</label>
-                <textarea id="marketAddress" class="form-control" v-model.trim="newShop.address" type="text" placeholder="Adresse des Supermarkts">
-                </textarea>
-            </div>
-        </div>
+  <div v-if="success" class="alert alert-success">
+    Markt erfolgreich eingetragen
```

Comment:
This could be `{{ success }}`, which is set to this string below, right?

Reply:
Success is just a book now